crypto/ec/curve25519.c

```diff
@@ -5570,6 +5632,14 @@ ossl_ed25519_verify(const uint8_t *message, size_t message_len,
     ge_tobytes(rcheck, &R);
 
     res = CRYPTO_memcmp(rcheck, r, sizeof(rcheck)) == 0;
+
+    /* note that we have used the strict verification equation here.
```

Comment:
The following notation is used elsewhere..
````
/*
 * Blah blah
 * Blah blah2
 */
 ````

Reply:
fixed.